.github/workflows/black.yml

```diff
@@ -4,8 +4,18 @@ name: Python Black Formatting
 on:
   push:
     branches:
-      - grass*
+      - grass[0-9]+
+      - master
```

Comment:
We won't be updating the master branch, so it does not need to be here. It is about the time we delete that branch to avoid confusion (default branch is not master, but master is present).